src/cloud/env/variables-build.md

```diff
@@ -201,6 +201,21 @@ stage:
     SKIP_SCD: false
 ```
 
+### `SKIP_COMPOSER_DUMP_AUTOLOAD`
```

Comment:
The variables are in alphabetical order. This section should be before SKIP_SCD.